content/blog/2023/contain-root.md

```diff
@@ -0,0 +1,540 @@
+---
+title: Can We Query the Root Container?
+sub: The complexities of containment, overflow, and 'propagation'
+author: miriam
+date: 2023-07-04
```

Comment:
I don't know if we actually want to publish tomorrow, or hold off for Wednesday?